### PR TITLE
docs(ecosystem): add simple-tjscli (#4112)

### PR DESCRIPTION
**Pull request by imjuni**
* Update docs/Guides/Ecosystem.md

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/Guides/Ecosystem.md [hunk 1]
```diff
@@ -557,3 +557,4 @@ section.
 #### [Community Tools](#community-tools)
 - [`fast-maker`](https://github.com/imjuni/fast-maker) route configuration
   generator by directory structure.
+- [`simple-tjscli`](https://github.com/imjuni/simple-tjscli) typescript interface convert json-schema.
```
Comment (jsumners) [inline docs/Guides/Ecosystem.md:560]: I think this description can be improved. As it is, it is a fragmented sentence that doesn't give a clear indication of what the library is.

Comment (imjuni) [reply docs/Guides/Ecosystem.md:560]: How about it?

```
cli tool to json-schema generate from typescript interface
```

Comment (jsumners) [reply docs/Guides/Ecosystem.md]: > cli tool to json-schema generate from typescript interface

"CLI tool to generate JSON Schema from TypeScript interfaces."

Comment (imjuni) [reply docs/Guides/Ecosystem.md]: @jsumners Thank you for your help! I changed sentence.

**Review by imjuni — COMMENTED** [review]
Thank you :)

**Review by mcollina — APPROVED** [review]
lgtm

**Review by jsumners — APPROVED** [review]

**Pull request merged**